xml/System/BitConverter.xml

```diff
@@ -809,9 +809,9 @@
         <Parameter Name="value" Type="System.Int32" Index="0" FrameworkAlternate="netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="value">The integer to convert.</param>
+        <summary>Converts the specified integer into a float.</summary>
```

Comment:
```suggestion
        <summary>Reinterprets the specified integer as a float.</summary>
```

Reply:
Also, I would change the existing docs for Int64BitsToDouble to also use "reinterpret".

"Conversion" typically means a simple cast, e.g. what you get from `(float)value`.

Reply:
Or we can say: "Converts the specified 32-bit pattern into a float."